scripts/dataconnect-test/tests.ts

```diff
@@ -28,45 +63,128 @@ async function list() {
   );
 }
 
-async function migrate() {
+async function migrate(force: boolean) {
+  const args = force ? ["--force"] : [];
+  if (FIREBASE_DEBUG) {
+    args.push("--debug");
+  }
   return await cli.exec(
     "dataconnect:sql:migrate",
     FIREBASE_PROJECT,
-    ["--force"],
+    args,
     __dirname,
     /** quiet=*/ false,
     { FIREBASE_CLI_EXPERIMENTS: "dataconnect" },
   );
 }
 
-async function deploy() {
-  return await cli.exec(
-    "deploy",
-    FIREBASE_PROJECT,
-    ["--only", "dataconnect", "--force"],
-    __dirname,
-    /** quiet=*/ false,
-    { FIREBASE_CLI_EXPERIMENTS: "dataconnect" },
-  );
+async function deploy(force: boolean) {
+  const args = ["--only", "dataconnect"];
+  if (force) {
+    args.push("--force");
+  }
+  if (FIREBASE_DEBUG) {
+    args.push("--debug");
+  }
+  return await cli.exec("deploy", FIREBASE_PROJECT, args, __dirname, /** quiet=*/ false, {
+    FIREBASE_CLI_EXPERIMENTS: "dataconnect",
+  });
+}
+
+function toPath(p: string) {
+  return path.join(__dirname, p);
+}
+
+function getRandomString(length: number): string {
```

Comment:
surprised that this doesn't have a native util

Reply:
we could use crypto.getRandomByes and cast it to a string, but crypto is not available in all node builds (https://nodejs.org/api/crypto.html#determining-if-crypto-support-is-unavailable) and I generally try to avoid using it